client/e2e/support/add-user.po.ts

```diff
@@ -0,0 +1,47 @@
+export interface TestUser {
+  name: string;
+  age: string;
+  company?: string;
+  email?: string;
+  role: 'admin' | 'editor' | 'viewer';
+}
+
+export class AddUserPage {
+  navigateTo() {
+    return cy.visit('/users/new');
+  }
+
+  getTitle() {
+    return cy.get('.add-user-title');
+  }
+
+  addUserButton() {
+    return cy.contains('button', 'ADD USER');
+  }
+
+  typeInput(inputId: string, text: string) {
+    const input = cy.get(`#${inputId}`).click();
```

Comment:
A think a comment here about string interpolation would probably be a Good Idea.